Documentation and bug fixes (#196)
AY2021S1-CS2113-T13-1/tp: docs/UserGuide.md

5. Use the command `help` to get a list of commands and their usages before using the application (if needed).

> 💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS Terminal and linux. Other terminals, such as Cygwin may not support color display.

Reviewer: ```suggestion
💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS and Linux Terminals. Other terminals, such as Cygwin may not support color display.
```

5. Use the command `help` to get a list of commands and their usages before using the application (if needed).

> 💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS Terminal and linux Terminal. Other terminals, such as Cygwin may not support color display.

Reviewer: ```suggestion
💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS Terminal, and Linux Terminal. Other terminals, such as Cygwin may not support color display.
```

Author: I updated this. Will ask for re-review when I complete all changes.

Format: `create-t /tag TAG [TAG_COLOR] [/tag TAG]...`

- Creates a tag with the name TAG. TAG can only be a single word.

> - [TAG_COLOR] is an optional input to specify the tag color to display. The possible colors are red, green, blue, yellow, purple, cyan, white. Any other input will set the tag as white. If no color is specified, the color of the tag will be set to the default white color. This applies to other input that has `/tag TAG [TAG_COLOR]` as an optional field. 

Reviewer: ```suggestion
- [TAG_COLOR] is an optional input to specify the tag color to display. The possible colors are red, green, blue, yellow, purple, cyan, white. If no color is specified, or an invalid input is entered, the color of the tag will be set to the default white color. This applies to all other inputs that have `/tag TAG [TAG_COLOR]` as an optional field as well. 
```

Format: `delete-e INDEX`

> - Deletes the event at the specified INDEX. The index refers to the index number shown in the displayed event list (list-e). The index must be a **positive integer** (1, 2, 3, …).

Reviewer: Why was this change made ah? I don't think there is a difference to the output right?

Author: Just for standardization since all are using '-'
